pyvista/plotting/plotting.py

```diff
@@ -3383,21 +3383,53 @@ def open_movie(self, filename, framerate=24, quality=5, **kwargs):
             filename = os.path.join(pyvista.FIGURE_PATH, filename)
         self.mwriter = get_writer(filename, fps=framerate, quality=quality, **kwargs)
 
-    def open_gif(self, filename):
+    def open_gif(self, filename, loop=0, fps=10, palettesize=256, subrectangles=False, **kwargs):
         """Open a gif file.
 
         Parameters
         ----------
         filename : str
             Filename of the gif to open.  Filename must end in ``"gif"``.
 
+        loop : int, optional
+            The number of iterations. Default 0 (meaning loop indefinitely).
+
+        fps : float, optional
+            The number of frames per second. If duration is not given, the
+            duration for each frame is set to 1/fps. Default 10.
```

Comment:
There's mention of "if duration is not given", but `duration` is not an explicit kwarg. Perhaps it should be? Or we should make it explicit that this can be passed on to `get_writer`?  And can we give a better pointer for `kwarg` documentation than `imageio.get_writer()`? (See also my comment for https://github.com/pyvista/pyvista/pull/2489)